docs/manual/java/releases/Migration15.md

```diff
@@ -46,30 +46,78 @@ were never intended for public consumption, and therefore have been marked depre
+See Deployment below.
 
-If you are not using Kubernetes or DC/OS, you must configure your services in a way that suits your production environment. See [[Running Lagom in production|ProductionOverview]] for more information.
+## Lightbend Orchestration
 
-If you are using ConductR in production and need assistance migrating to other solutions please [contact Lightbend](https://www.lightbend.com/contact).
+[Lightbend Orchestration](https://developer.lightbend.com/docs/lightbend-orchestration/current/) is not supported with Lagom 1.5. If you used Lightbend Orchestration for your deployment you will have to migrate to a manual process. This migration may happen on the same commit where you upgrade to Lagom 1.5.0 or you can upgrade your deployment process on a first step (still using Lagom 1.4.x) and later upgrade to Lagom 1.5.0.
 
-## TLS Support
+See Deployment below.
 
-## Lightbend Orchestration
+## Deployment
 
-TODO (new version required)
+ConductR tooling and Lightbend Orchestration handled all the required pieces to deploy on ConductR and Kubernetes or DC/OS. Lagom 1.5.0 only supports a manually maintained deployment process.
 
-## Cluster Formation
+In particular, ConductR tooling and Lightbend Orchestration handled some or all of the following:
 
-A new mechanism to form and [[join an Akka Cluster|Cluster#Joining]] is introduced in Lagom 1.5. Apart from the original `Manual Cluster Formation` the new `Akka Cluster Bootstrap` is now supported. This new mechanism is introduced with lower precedence than `Manual Cluster Formation` so if you rely on the use of a list of `seed-nodes` then everything will work as before. On the other hand, `Akka Cluster Bootstrap` takes precedence over the `join-self` cluster formation for single-node clusters. If you use single-node clusters via `join-self` you will have to explicitly disable `Akka Cluster Bootstrap`:
+1. extend the your application with: cluster bootstrapping, akka management and health checks
+2. Service Location
+3. setup and produce docker images
+4. prepare the deployment specs for the target orchestrator
+5. Secrets
+
+#### Application extensions
+
+Starting with Lagom 1.5 your application will include [[Akka management HTTP|Cluster#Akka-Management]] out of the box with [[health checks|Cluster#Health-Checks]] enabled by default. Cluster formation also supports [[Cluster Bootstrapping|Cluster#Joining-during-production-(Akka-Cluster-Bootstrap)]] as a new way to form a cluster. These new defaults may require at least two changes on your codebase. First, if you want to opt-in to cluster bootstrapping you must make sure you don't set `seed-nodes`. Second, if you use Cluster Bootstrapping, you will have to setup a [[discovery|Cluster#Akka-Discovery]] mechanism (see the [[reference guide|Cluster#Akka-Discovery]] for more details). 
```

Comment:
> Starting with Lagom 1.5 your application will include [[Akka management HTTP|Cluster#Akka-Management]] out of the box with [[health checks|Cluster#Health-Checks]] enabled by default.

Maybe explain that Akka Management HTTP is included _in order_ to define health checks for the cluster.  Otherwise it remains unclear why it's added (IMO).